common/params.cc

```diff
@@ -202,7 +202,7 @@ std::unordered_map<std::string, uint32_t> keys = {
     {"Offroad_TemperatureTooHigh", CLEAR_ON_MANAGER_START},
     {"Offroad_UnofficialHardware", CLEAR_ON_MANAGER_START},
     {"Offroad_UpdateFailed", CLEAR_ON_MANAGER_START},
-    {"Offroad_Recalibration", CLEAR_ON_ONROAD_TRANSITION},
+    {"Offroad_Recalibration", CLEAR_ON_MANAGER_START | CLEAR_ON_ONROAD_TRANSITION},
```

Comment:
Want me to add a test to assert `CLEAR_ON_MANAGER_START` if `CLEAR_ON_ONROAD_TRANSITION`?  Since things can break if you reboot the device onroad now that we don't clear in boardd.

Reply:
no, we should just also clear them at the same time

Reply:
https://github.com/commaai/openpilot/pull/28198